### PR TITLE
Libsndfile

### PR DESCRIPTION
**Pull request by stefan-balke**
Duplicated libsamplerate to compile libsndfile.

### CODE REVIEW

**Comment by stefan-balke** [comment]
I'm missing autogen here and I will create a recipe for that.

Is the correct way to fork conda/conda-recipes and make a PR there?

**Comment by ocefpaf** [comment]
> Is the correct way to fork conda/conda-recipes and make a PR there?

If you want to contribute to `conda/conda-recipes` too yes. But we do not have a direct relationship to that repository, so you can send a PR for `autogen` here directly.

**Comment by stefan-balke** [comment]
Alright, so if I do "conda install" without any further repos, where do the recipes come from? Can we place it there?

**Comment by ocefpaf** [comment]
> Alright, so if I do "conda install" without any further repos, where do the recipes come from? Can we place it there?

That is the default channel. To have this package in the default channel you need to make a request to Continuum. The `conda-forge` channel is a community channel, so you will  be able to `conda install -c conda-forge <your_package>`. Also, if your package is popular here, in the community channel, you'll have a stronger case to get it into the default channel.

PS: Note that the `conda/conda-recipes` repository is just a recipe repository. No binary gets build. I believe that, eventually, continuum folks look there and use some of the recipes to augment the default channel, but there is no direct relationship. We do hope that conda-forge will have a closer relationship with the default channel.

**Comment by stefan-balke** [comment]
Okay, gotcha. Don't know yet if this lib is worth it since the dependencies sum up...

**Comment by ocefpaf** [comment]
> Okay, gotcha. Don't know yet if this lib is worth it since the dependencies sum up...

We recommend you to get in touch with people that are interested in having this software installable via conda and invite them to be maintainers of the recipes. That way you reduce the burden on your shoulders. Most of conda-forge people will help you with the initial recipe review, but after that you'll need to curate a community around your `feedstocks`. (The `feedstocks` are the repositories where the recipes lives after being merged here).

**Comment by stefan-balke** [comment]
There certainly are people interested in this.

The motivation is to an anaconda package for https://github.com/bmcfee/librosa/issues/202
which is cross-platform.

**Comment by ocefpaf** [comment]
> There certainly are people interested in this.

Awesome! Bring them to conda-forge :wink:

File: recipes/libsndfile/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    - autoconf
+    - autogen
+    - automake
+    - libtool
+    - pkg-config
+    - python
+
+test:
+  commands:
+    - true  # we perform testing in the build phase
```
Comment (ocefpaf) [inline recipes/libsndfile/meta.yaml]: Ping @stefan-balke

File: recipes/libsndfile/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+about:
+  home: http://www.mega-nerd.com/libsndfile/
+  license: GPL
```
Comment (jakirkham) [inline recipes/libsndfile/meta.yaml]: Version? Says [LGPL 3]( http://www.mega-nerd.com/libsndfile/ ).

File: recipes/libsndfile/bld.bat [hunk 3]
```diff
@@ -0,0 +1 @@
+Make.bat
```
Comment (jakirkham) [inline recipes/libsndfile/bld.bat]: Where is this coming from?

File: recipes/libsndfile/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  number: 0
+
+requirements:
+  build:
+    - autoconf
+    - autogen
+    - automake
+    - libtool
+    - pkg-config
+    - python
```
Comment (jakirkham) [inline recipes/libsndfile/meta.yaml]: If we are doing Windows, most of these need UNIX selectors.

File: recipes/libsndfile/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,34 @@
+{% set version = "1.0.26" %}
+
+package:
+  name: libsndfile
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/erikd/libsndfile.git
+  git_tag: f67f0992eae3d19ca3a3804c8ea6b081b2131e5f
```
Comment (jakirkham) [inline recipes/libsndfile/meta.yaml]: Can we download a tarball instead? They have some on their [site]( http://www.mega-nerd.com/libsndfile/#Download ).

**Comment by stefan-balke** [comment]
For the reference: http://clam-project.org/wiki/Devel/Windows_MinGW_cross_compile#libsndfile

**Comment by jakirkham** [comment]
Any thoughts on the proposed changes?

**Comment by stefan-balke** [comment]
> Any thoughts on the proposed changes?

Haven't touched this since the dependencies are not done yet.

Is there any recipe with bundled DLLs for Windows?

**Comment by scopatz** [comment]
ping @stefan-balke, any movement on this?

**Comment by isuruf** [comment]
Already in conda-forge

**Pull request closed without merging**